### PR TITLE
Config updates to be reviewed

### PR DESCRIPTION
**Pull request by infojunkie**
@danielafeitosa @caiosba plz review changes here, including:
- updated example files (including comments)
- delete files
- there are a few `.example` files that have no equivalent in `configurator` - can these be deleted too? `secret_token.rb.example` and `profile.rb.example`

### CODE REVIEW

File: config/config.yml.example [hunk 1]
```diff
@@ -1,40 +1,88 @@
...
+  # We use Luminati https://luminati.io/
+  #
+  # OPTIONAL
+  #
+  proxy_host: '<LUMINATI HOST>'
+  proxy_port: '<LUMINATI PORT>'
+  proxy_user_prefix: '<LUMINATI ACCOUNT PREFIX>'
+  proxy_pass: '<LUMINATI ACCOUNT PASSWORD>'
+
+  # Google API to query YouTube posts
```
Comment (danielafeitosa) [inline config/config.yml.example]: It's also used to check malicious websites using Lookup API (optional)

File: config/config.yml.example [hunk 2]
```diff
@@ -1,40 +1,88 @@
...
+  # Twitter API to query Twitter posts
+  #
+  # REQUIRED for Twitter posts
+  #
+  twitter_consumer_key: '<TWITTER APP CONSUMER KEY>'
+  twitter_consumer_secret: '<TWITTER APP CONSUMER SECRET>'
+  twitter_access_token: '<TWITTER APP ACCESS TOKEN>'
+  twitter_access_token_secret: '<TWITTER APP ACCESS TOKEN SECRET>'
+
+  # Facebook API to query Facebook posts
```
Comment (danielafeitosa) [inline config/config.yml.example]: We are not using to parse the FB posts anymore. We use it to get engagement metrics on Facebook for any URL. If the config is missing we only won't be able to get the metrics on FB.

**Comment by danielafeitosa** [comment]
>     * there are a few `.example` files that have no equivalent in `configurator` - can these be deleted too? `secret_token.rb.example` and `profile.rb.example`

The `profile.rb.example` is useful to analyze when the requests are slow. `secret_token.rb.example` is not needed for `development` environment but it's used on `production` environment. I'm not sure if the file should be deleted

**Comment by danielafeitosa** [comment]
> why do we have those Sidekiq queues in `configurator` for Pender local

We don't need these queues on `configurator`. Probably it was a bad copy/paste.

File: config/config.yml.example [hunk 3]
```diff
@@ -1,40 +1,88 @@
...
+    video_asset_path: 'http://localhost:9000/pender-video/video'
+
+  # Cache-clearing service
+  #
+  # SOON-TO-BE-DEPRECATED
+  # Use values below
+  #
+  cc_deville_host: https://cc-aws.meedan.com
+  cc_deville_token: e1CXbbwt6s1tY8VsglaxxRbF6ghivU5o
+  cc_deville_httpauth:
```
Comment (caiosba) [inline config/config.yml.example]: We don't need CC Deville locally, only in production... and the application won't crash if those settings are blank... so I suggest to leave them blank.

File: config/config.yml.example (hunk 1 above)
Comment (infojunkie) [reply config/config.yml.example]: Can you plz give me a link to Lookup API documentation? Also what is the process of getting a Google API key (if you know)

File: config/config.yml.example (hunk 2 above)
Comment (infojunkie) [reply config/config.yml.example]: thanks

File: config/config.yml.example [hunk 4]
```diff
@@ -1,40 +1,88 @@
...
+  # Special HTTP headers to access specific hosts
+  #
+  # OPTIONAL
+  #
   hosts:
-    one.host:
-      cf_credentials: 1234:5678
-    yet.another.host:
-      country: cc
+    <HOST DOMAIN>:
```
Comment (infojunkie) [inline config/config.yml.example]: @caiosba @danielafeitosa is this doc correct ^ ?

Comment (caiosba) [reply config/config.yml.example]: no... we currently use it only for country proxies and for Cloudflare credentials

Comment (caiosba) [reply config/config.yml.example]: this means that we add a `host` mapped to a `country`, we will use a proxy from that country to parse URLs from that domain

File: config/config.yml.example (hunk 1 above)
Comment (danielafeitosa) [reply config/config.yml.example]: The Lookup API: https://developers.google.com/safe-browsing/v4/lookup-api
Here it says the first steps and how to get a Google API key: https://developers.google.com/safe-browsing/v4/get-started

**Pull request merged**